examples/main/main.cpp

```diff
@@ -1012,7 +1034,7 @@ int main(int argc, char ** argv) {
             // output to JSON file
             if (params.output_jsn) {
                 const auto fname_jsn = fname_out + ".json";
-                output_json(ctx, fname_jsn.c_str(), params, pcmf32s);
+                output_json(ctx, fname_jsn.c_str(), params, pcmf32s, true);
```

Comment:
Could you offer an alternative option, allowing users to decide if they require token-level data?